### PR TITLE
🧼  NLP task guides

### PR DESCRIPTION
**Pull request by stevhliu**
A clean commit of the NLP task guides #15564 😭

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
LGTM! Feel free to merge once @sgugger approves

File: docs/source/tasks/question_answering.mdx
Comment (flozi00) [inline docs/source/tasks/question_answering.mdx:26]: Is that link correct?

File: docs/source/_toctree.yml
Comment (sgugger) [inline docs/source/_toctree.yml:156]: This should not be removed.

File: docs/source/_toctree.yml
Comment (sgugger) [inline docs/source/_toctree.yml:252]: This not either.

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:20]: Maybe mention another tokenizer/architecture can be used the same way somewhere? This feels like it only works for DistilGPT2/DistilRoBERTa

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:173]: The task video should be linked in the intro IMO. It doesn't show anything about training.

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:189]:
```suggestion
If you aren't familiar with fine-tuning a model with the [`Trainer`], take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:234]: This should all go in the same snippet as for PyTorch, using the PT/TF split.

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:288]: Same comment as above.

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:304]:
```suggestion
If you aren't familiar with fine-tuning a model with the `Trainer`, take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/language_modeling.mdx
Comment (sgugger) [inline docs/source/tasks/language_modeling.mdx:350]: Should also be in the same snippet as PyTorch, with the PT/TF split.

File: docs/source/tasks/multiple_choice.mdx
Comment (sgugger) [inline docs/source/tasks/multiple_choice.mdx:145]:
```suggestion
If you aren't familiar with fine-tuning a model with the `Trainer`, take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/question_answering.mdx
Comment (sgugger) [reply docs/source/tasks/question_answering.mdx:26]: +1, wrong copy-paste here ;-)

File: docs/source/tasks/question_answering.mdx
Comment (sgugger) [inline docs/source/tasks/question_answering.mdx:155]:
```suggestion
If you aren't familiar with fine-tuning a model with the `Trainer`, take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/question_answering.mdx
Comment (sgugger) [inline docs/source/tasks/question_answering.mdx:204]: Like before, should be in the same snippet as PyTorch.

File: docs/source/tasks/sequence_classification.mdx
Comment (sgugger) [inline docs/source/tasks/sequence_classification.mdx:95]:
```suggestion
If you aren't familiar with fine-tuning a model with the [`Trainer`], take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/sequence_classification.mdx
Comment (sgugger) [inline docs/source/tasks/sequence_classification.mdx:149]: Should be in the same snippet as PyTorch.

File: docs/source/tasks/summarization.mdx
Comment (sgugger) [inline docs/source/tasks/summarization.mdx:114]:
```suggestion
If you aren't familiar with fine-tuning a model with the `Trainer`, take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/summarization.mdx
Comment (sgugger) [inline docs/source/tasks/summarization.mdx:165]: Should be in the same snippet as PyTorch.

File: docs/source/tasks/token_classification.mdx
Comment (sgugger) [inline docs/source/tasks/token_classification.mdx:155]:
```suggestion
If you aren't familiar with fine-tuning a model with the `Trainer`, take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/token_classification.mdx
Comment (sgugger) [inline docs/source/tasks/token_classification.mdx:204]: Should go in the same snippet as PyTorch.

File: docs/source/tasks/translation.mdx
Comment (sgugger) [inline docs/source/tasks/translation.mdx:116]:
```suggestion
If you aren't familiar with fine-tuning a model with the [`Trainer`], take a look at the basic tutorial [here](training#finetune-with-trainer)!
```

File: docs/source/tasks/translation.mdx
Comment (sgugger) [inline docs/source/tasks/translation.mdx:167]: Should be in the same snippet as PyTorch.

**Review by sgugger — APPROVED** [review]
Thanks for adding all tasks! I left a few more comments.

File: docs/source/tasks/question_answering.mdx
Comment (stevhliu) [reply docs/source/tasks/question_answering.mdx:26]: Oops, thanks for catching this!

File: docs/source/tasks/language_modeling.mdx
Comment (stevhliu) [reply docs/source/tasks/language_modeling.mdx:173]: Linked both task videos in the intro

**Pull request merged**
